### PR TITLE
Adopt CakePHP CD for cake.css

### PR DESCRIPTION
**Pull request by cleptric**
I updated the cake.css file like proposed in #481
My "concept" was to simplify the basic color palette a bit.
The main color now is #D33C43, with a 10% darker and a 10% lighter accent alternative.
I also added the cake logo to the header.
I would call this a first step, because there are still some other things to improve.

Before:

screen shot 2017-01-18 at 21 54 13

Now:

screen shot 2017-01-18 at 23 14 48

EDIT: Now shows state of the last commit

### CODE REVIEW

**Comment by josegonzalez** [comment]
The red links look super aggressive. Can we use the same colors from the book?

**Comment by cleptric** [comment]
Sure. We could also style the body in different shades of grey (not kidding) and just use the red for the header.

File: webroot/css/cake.css [hunk 1]
```diff
@@ -350,20 +350,33 @@ pre {
 }
 
 .top-bar.expanded .title-area {
-    background: #01545b;
+    background: #ba232a;
+}
+
+.top-bar.expanded .title-area:before {
```
Comment (markstory) [inline webroot/css/cake.css]: Any way we could reduce the specificity here?

Comment (cleptric) [inline webroot/css/cake.css]: `.title-area:before` is fine, too. 
Just out of curiosity, is less specific best practice?

**Comment by phpnut** [comment]
I do have new designs coming for app/. I forgot to get them with the redesign. Don't get discouraged when your hard work here gets replaced. Thanks for you effort and it is looking good.

File: webroot/css/cake.css [hunk 2]
```diff
@@ -553,4 +566,4 @@ div.message.hidden {
 table td {
     vertical-align: top;
     word-break: break-all;
-}
+}
```
Comment (dereuromark) [inline webroot/css/cake.css]: As a side note: Make sure your IDE is properly configured (ending newline).

**Comment by josegonzalez** [comment]
@cleptric The links on the side should be something other than black, to make them more "link-like" :)

**Comment by cleptric** [comment]
@josegonzalez Use the colors of the book he said 😂 That are the same colors as the links in the sidebar of the book. But I'll change them anyway 😀

**Comment by josegonzalez** [comment]
Oh nvm I'm a noob. Carry on then!

**Comment by Oxicode** [comment]
> The red links look super aggressive

Its tue

**Comment by josegonzalez** [comment]
@phpnut is this still waiting on new designs?

**Comment by cleptric** [comment]
I'll close this for now. When we have the new design we can reopen it 🙂

**Pull request closed without merging**